contracts/SpokePool.sol

```diff
@@ -286,7 +321,54 @@ abstract contract SpokePool is Testable, Lockable, MultiCaller {
         uint256 relayerRefundId,
         MerkleLib.DestinationDistribution memory distributionLeaf,
         bytes32[] memory inclusionProof
-    ) public {}
+    ) public override {
+        // Check integrity of leaf structure:
+        require(distributionLeaf.chainId == chainId(), "Invalid chainId");
+        require(distributionLeaf.refundAddresses.length == distributionLeaf.refundAmounts.length, "invalid leaf");
+
+        // Grab distribution root stored at `relayerRefundId`.
+        RelayerRefund storage refund = relayerRefunds[relayerRefundId];
+
+        // Check that `inclusionProof` proves that `distributionLeaf` is contained within the distribution root.
+        // Note: This should revert if the `distributionRoot` is uninitialized.
+        require(
+            MerkleLib.verifyRelayerDistribution(refund.distributionRoot, distributionLeaf, inclusionProof),
```

Comment:
nit: if you re-name the `inclusionProof` to `proof` t his might fit on one line and then it's consistant with the `HubPool`.

Reply:
done